public/scripts.js

```diff
@@ -53,6 +66,62 @@ document.addEventListener('DOMContentLoaded', function() {
     }, 1000);
   }
 
+  function showTalkingHead(author, quote) {
+    var img = new Image();
+    var author = author.toLowerCase();
+
+    img.onload = function() {
+      canvas.style.visibility = 'visible';
+      makeAuthorTalk(this, author, quote);
+    };
+    img.src = 'images/faces/' + author.toLowerCase() + '.png';
+  }
+
+  function muteTalkingHead() {
+    clearInterval(canvasInterval);
+  }
+
+  function hideTalkingHead() {
+    muteTalkingHead();
+    canvas.style.visibility = 'hidden';
```

Comment:
Yeah, I guess it's kinda necessary. At least in current implementation. I added those `visibility` to make it look smooth when quote is changing, basically I just hide it when quote is changing and head doesn't hang alone on the page.